src/main/java/io/cdap/plugin/gcp/bigquery/util/BigQueryConstants.java

```diff
@@ -42,4 +42,5 @@
   String CONFIG_PARTITION_INTEGER_RANGE_START = "cdap.bq.sink.partition.integer.range.start";
   String CONFIG_PARTITION_INTEGER_RANGE_END = "cdap.bq.sink.partition.integer.range.end";
   String CONFIG_PARTITION_INTEGER_RANGE_INTERVAL = "cdap.bq.sink.partition.integer.range.interval";
+  String CONFIG_TEMPORARY_TABLE_NAME = "cdap.source.temporary.table.name";
```

Comment:
should follow same pattern of string as above

Reply:
Changed to: `String CONFIG_TEMPORARY_TABLE_NAME = "cdap.bq.source.temporary.table.name";`